docs/spark-connector/spark-catalog-iceberg.md

```diff
@@ -97,44 +97,29 @@ DESC EXTENDED employee;
+| `jdbc-password`                 | `jdbc.password`                       | JDBC password                                                                                                                                                                                                       | 0.5.0         |
+| `io-impl`                       | `io-impl`                             | The io implementation for `FileIO` in Iceberg.                                                                                                                                                                      | 0.6.0         |
+| `s3-endpoint`                   | `s3.endpoint`                         | An alternative endpoint of the S3 service, This could be used for S3FileIO with any s3-compatible object storage service that has a different endpoint, or access a private S3 endpoint in a virtual private cloud. | 0.6.0         | 
+| `s3-region`                     | `client.region`                       | The region of the S3 service, like `us-west-2`.                                                                                                                                                                     | 0.6.0         |
 
-#### RESTCatalog
-
-| Gravitino catalog property name | Spark Iceberg connector configuration | Default Value | Required | Description               | Since Version |
-|---------------------------------|---------------------------------------|---------------|----------|---------------------------|---------------|
-| `catalog-backend`               | `type`                                | `memory`      | Yes      | Catalog backend type      | 0.5.1         |
-| `uri`                           | `uri`                                 | (none)        | Yes      | Catalog backend uri       | 0.5.1         |
-| `warehouse`                     | `warehouse`                           | (none)        | No       | Catalog backend warehouse | 0.5.1         |
-
-Gravitino catalog property names with the prefix `spark.bypass.` are passed to Spark Iceberg connector. For example, using `spark.bypass.io-impl` to pass the `io-impl` to the Spark Iceberg connector.
+Gravitino catalog property names with the prefix `spark.bypass.` are passed to Spark Iceberg connector. For example, using `spark.bypass.clients` to pass the `clients` to the Spark Iceberg connector.
 
 :::info
 Iceberg catalog property `cache-enabled` is setting to `false` internally and not allowed to change.
 :::
 
+## Storage
+
+### S3
+
+To access the table stored on S3, you must add s3 secret in Spark configuration by `spark.sql.catalog.${iceberg_catalog_name}.s3.access-key-id` and `spark.sql.catalog.${iceberg_catalog_name}.s3.secret-access-key`, download [Iceberg AWS bundle](https://mvnrepository.com/artifact/org.apache.iceberg/iceberg-aws-bundle) and place it to the classpath of Spark.
```

Comment:
the table -> tables.

> you must add s3 secret in Spark configuration by `spark.sql.catalog.${iceberg_catalog_name}.s3.access-key-id` and `spark.sql.catalog.${iceberg_catalog_name}.s3.secret-access-key`

You need to add s3 ... to the spark configuration using ... and .....,  additionally, download the ... and place it in spark's classpath.